nav2_map_server/src/map_saver/main_cli.cpp

```diff
@@ -163,11 +193,26 @@ int main(int argc, char ** argv)
   // Call saveMapTopicToFile()
   int retcode;
```

Comment:
If you set recode here to 1, then you only need to set to `0` if successful and then can remove both of the `else` statements.